tensorflow/core/common_runtime/direct_session_with_tracking_alloc_test.cc

```diff
@@ -101,11 +101,27 @@ TEST(DirectSessionWithTrackingAllocTest, CostModelTest) {
         EXPECT_EQ(2, shape.dim_size());
         EXPECT_EQ(2, shape.dim(0).size());
         EXPECT_EQ(1, shape.dim(1).size());
+#ifdef INTEL_MKL
```

Comment:
It would be better to duplicate only constant values rather than the whole if statement